config/routes/admin.rb

```diff
@@ -95,6 +95,8 @@
       end
     end
 
+    resource :forwarding, only: [:new, :create], controller: 'reports/forward'
```

Comment:
I'm not super keen on these routes:

-`GET /admin/reports/18/forwarding/new`
- `POST /admin/reports/18/forwarding/`

Open to suggestions.

Reply:
`resource :forward, only: [:new, :create]` ?